registry/extension/oras/oras.go

```diff
@@ -88,11 +88,12 @@ func (d *orasNamespace) GetRepositoryRoutes() []extension.Route {
 			Extension: extensionName,
 			Component: referrersComponentName,
 			Descriptor: v2.RouteDescriptor{
-				Entity: "Referrers",
+				Entity:      "Referrers",
+				Description: "returns all referrers for a given digest",
 				Methods: []v2.MethodDescriptor{
 					{
 						Method:      "GET",
-						Description: "Get all referrers for the given digest. Currently the API doesn't support pagination.",
+						Description: "Get all referrers for the given digest",
```

Comment:
I'm a bit confused on when should we capitalize the first letter in the description, and when shouldn't?

Reply:
good point. I'm not sure either. I'll switch it to lower case since the other route descriptions are lower case